ddsp/ai-platform/README.md

```diff
@@ -0,0 +1,103 @@
+
+Docker image for training autoencoder on [Google Cloud AI Platform](https://cloud.google.com/ai-platform).
+
+### Before you begin
+Make sure that you have completed the following steps:
+* Set up your [GCP project](https://cloud.google.com/resource-manager/docs/creating-managing-projects)
+* Create a [Google Cloud Storage Bucket](https://cloud.google.com/storage/docs/creating-buckets)
+* Enable [AI Platform Training and Prediction, Container Registry, and Compute Engine APIs](https://pantheon.corp.google.com/flows/enableapi?apiid=ml.googleapis.com,compute_component,containerregistry.googleapis.com)
+* Install [Docker](https://docs.docker.com/engine/install/)
+* [Configure Docker for Cloud Container Registry](https://cloud.google.com/container-registry/docs/pushing-and-pulling)
+* [Upload the training data](https://cloud.google.com/storage/docs/uploading-objects) in the [TFRecord](https://www.tensorflow.org/tutorials/load_data/tfrecord) format to the GCS bucket. You can preprocess your audio files into this format using the `ddsp_prepare_tfrecord` tool as described in [Making a TFRecord dataset from your own sounds](https://github.com/magenta/ddsp/tree/master/ddsp/training/data_preparation).
+
+### Quickstart:
+
+#### Define some environment variables
+
+We recommend setting `$REGION` accordingly to your location. We also recommend to [setup hostname](https://cloud.google.com/container-registry/docs/pushing-and-pulling#tag_the_local_image_with_the_registry_name) in `$IMAGE_URI` based of the `$REGION` choice as if your Docker images are stored in different region than the job is computed additional charges will be applied. 
+
+```bash
+export PROJECT_ID=[YOUR_PROJECT_ID]
+export SAVE_DIR=gs://[YOUR_STORAGE_BUCKET_NAME]/[PATH_IN_STORAGE_BUCKET]
+export FILE_PATTERN=gs://[YOUR_STORAGE_BUCKET_NAME]/[PATH_IN_STORAGE_BUCKET]/train.tfrecord*
+export IMAGE_REPO_NAME=ddsp_train
+export IMAGE_TAG=ai_platform
+export IMAGE_URI=gcr.io/$PROJECT_ID/$IMAGE_REPO_NAME:$IMAGE_TAG
+export REGION=us-central1
+export JOB_NAME=ddsp_container_job_$(date +%Y%m%d_%H%M%S)
+```
+#### Build the image and push it to Container Registry
+In the folder containing `Dockerfile` and `task.py` run following commands: 
+```bash
+docker build -f Dockerfile -t $IMAGE_URI ./
+docker push $IMAGE_URI
+```
+
+#### Submit the training to AI Platform
+```bash
+gcloud ai-platform jobs submit training $JOB_NAME \
+  --region $REGION \
+  --config config_single_vm.yaml \
+  --master-image-uri $IMAGE_URI \
+  -- \
+  --save_dir=$SAVE_DIR \
+  --file_pattern=$FILE_PATTERN \
+  --batch_size=16 \
+  --learning_rate=0.0001 \
+  --num_steps=40000 \
+  --early_stop_loss_value=5.0
+```
+##### AI Platform flags:
+* `--region` - Region when the training job is run
+* `--config` - Cluster configuration. In the example above, training on multiple VMs with multiple GPUs is set. For more information about various configurations take a look at **Note on cluster configuration and hyperparameters** below.
```

Comment:
seems like it's `config_single_vm` is this doc string correct?

Reply:
Yes, you're right. I had multiple VMs for a while in there, then changed it and didn't spotted that description needs to change as well. Thanks!

Reply:
Done